File: CRUDAccess/InvoiceRestAccess/Views/Model/PaymentHistoryView.cs
// ALLOWOVERWRITE-AF6240ECAC95061BC0BCD88AFA6E21FA

using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;


namespace Database.Views.Model
{

	[DataContract]
    public class PaymentHistoryView
    {
                       /// <summary>
                       /// 
                       /// </summary>
                       [DataMember(Name = "amount")]            
                       public decimal Amount { get; set; }	    	
                       /// <summary>
                       /// 
                       /// </summary>
                       [DataMember(Name = "paymentDate")]            
                       public DateTime PaymentDate { get; set; }	    	
                       /// <summary>
                       /// 
                       /// </summary>
                       [DataMember(Name = "paymentHistoryId")]            
                       public int PaymentHistoryId { get; set; }	    	
                       /// <summary>
                       /// 
                       /// </summary>
                       [DataMember(Name = "reference")]            
                       public string Reference { get; set; }	    	
    						  
					   /// <summary>
                      /// Reference of PrimaryKey for Invoice
                      /// </summary>
                      [DataMember(Name = "invoiceInvoiceId")]
					  public  int InvoiceInvoiceId { get; set; }   
					  
					  /// <summary>
                      /// Reference to  Invoice InvoiceNo Property
                      /// </summary>
				      [DataMember(Name = "invoiceInvoiceNo")]                      
					  public string InvoiceInvoiceNo { get; set; }   
    	
    	
    	
    	
   
    }
}

File: CRUDAccess/InvoiceWebClient/src/app/modules/admin/PaymentHistory/Models/PaymentHistoryView.ts
// ALLOWOVERWRITE-0085A2317930FAB9035BD26E1E682BC5


    export interface IPaymentHistoryView {
      amount: number;    	
      paymentDate: string;    	
      paymentHistoryId: number;    	
      reference: string;    	

					    invoiceInvoiceId : number;   
					    invoiceInvoiceNo : string;   
    }
    
    export class PaymentHistoryView implements IPaymentHistoryView {
      amount: number;    	
      paymentDate: string;    	
      paymentHistoryId: number;    	
      reference: string;    	

					    invoiceInvoiceId : number ;   
					    invoiceInvoiceNo : string;   


	  constructor() {
	  }
    }

